src/brainglobe_napari/tests/conftest.py

```diff
@@ -0,0 +1,48 @@
+import os
+from pathlib import Path
+
+import pytest
+from bg_atlasapi import config
+
+
+@pytest.fixture(autouse=True)
+def mock_brainglobe_user_folders(monkeypatch):
+    """Ensures user config and data is mocked during all local testing.
+
+    User config and data need mocking to avoid interfering with user data.
+    Mocking is achieved by turning user data folders used in tests into
+    subfolders of a new ~/.brainglobe-tests folder instead of ~/.
+
+    It is not sufficient to mock the home path in the tests, as this
+    will leave later imports in other modules unaffected.
+
+    GH actions workflow will test with default user folders.
+    """
+    if not os.getenv("GITHUB_ACTIONS"):
+        home_path = Path.home()  # actual home path
+        mock_home_path = Path(home_path / ".brainglobe-tests")
```

Comment:
```suggestion
        mock_home_path = home_path / ".brainglobe-tests"
```
Since home_path is already a Path shouldn't need this